radicle-cli/src/terminal/io.rs

```diff
@@ -56,9 +56,10 @@ pub fn tip_args(args: fmt::Arguments) {
     );
 }
 
-pub fn width() -> usize {
-    let (_, rows) = console::Term::stdout().size();
-    rows as usize
+pub fn width() -> Option<usize> {
+    console::Term::stdout()
+        .size_checked()
+        .map(|(_, cols)| cols as usize)
```

Comment:
What happens now if no width is returned? We continue as if its infinite?

Reply:
Yeah exactly. We just don't wrap lines. I think the reason this might be happening is when running the command not via a tty, eg. in the tests.